src/commands/commit/withClient/index.js

```diff
@@ -26,18 +26,27 @@ const withClient = async (answers: Answers) => {
 
     if (configurationVault.getAutoAdd()) await execa('git', ['add', '.'])
 
-    const { stdout } = await execa('git', [
+    const extraMessageArgs = answers.message
+      ? ['-m', `'${answers.message}'`]
+      : []
+
+    const args = [
       'commit',
       ...isSigned,
       '-m',
-      title,
-      '-m',
-      answers.message
-    ])
+      `'${title}'`,
+      ...extraMessageArgs
+    ]
 
-    console.log(stdout)
+    await execa('git', args, { stdio: 'inherit', shell: true })
   } catch (error) {
-    console.error(error)
+    const { exitCode, command } = error
```

Comment:
why not destruct in catch ? 

```js
} catch ({ exitCode, command }) {
```

Reply:
I think the intention was to print the error as is AND print a nice message. Destructing the error in catch would eat up any extra content in the error message.